modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
+= Use Studio to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To use Anypoint Studio to configure a connector:
+
+. Install the connector.
+. Configure an input source for the connector.
```

Comment:
I would say an "input source for the flow of the connector" or something like that. Connectors don't have input sources, flows do. Some connectors even have operations that can act as input sources for the flow.